index.html

```diff
@@ -2,9 +2,16 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
```

Comment:
As we discussed some time ago - it's better to keep the fonts in your project

Reply:
I removed links and installed roboto font via npm.